### PR TITLE
Use the reusable scripts from the same ref as the called reusable workflows

### PR DESCRIPTION
**Pull request by fbeutin-ledger**
Seen in action here
https://github.com/LedgerHQ/app-boilerplate/pull/52/checks

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,201 @@
...
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```
Comment (agrojean-ledger) [inline LICENSE:5]: Why the license change ?

File: .github/workflows/_check_icons.yml [hunk 2]
```diff
@@ -3,6 +3,9 @@ name: Check icons
 on:
   workflow_call:
     inputs:
+      ledger-app-workflows_ref:
```
Comment (agrojean-ledger) [inline .github/workflows/_check_icons.yml:6]: No default ?

**Comment by agrojean-ledger** [comment]
@fbeutin-ledger I find it a bit weird that `_get_workflow_version.yaml` is directly in  `.github/workflows` as it is not really for the user... :thinking:

File: .github/workflows/_check_icons.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (fbeutin-ledger) [reply .github/workflows/_check_icons.yml:6]: Done

**Comment by fbeutin-ledger** [comment]
> @fbeutin-ledger I find it a bit weird that `_get_workflow_version.yaml` is directly in `.github/workflows` as it is not really for the user... 🤔

There is no choice on this, all workflow related files are required to be in the `.github/workflows` dir
To mitigate the confusion the workflows are structure like this:
```
 _check_clang_static_analyzer.yml
 _check_icons.yml
 _check_makefile.yml
 _check_readme.yml
 _get_workflow_version.yaml
 reusable_build.yml
 reusable_guidelines_enforcer.yml
 reusable_lint.yml
 reusable_ragger_tests.yml
 scripts_checking.yml
```

- `_*` internal use, not a standalone workflow
- `reusable_*` reusable workflows available for users
- `xyz_*` standard CI workflows of this repo

**Review by agrojean-ledger — APPROVED** [review]

**Pull request merged**